src/pages/home/Home.tsx

```diff
@@ -1,88 +1,120 @@
-import React, { useMemo } from 'react';
+import React from 'react';
 import {
   IonContent,
   IonPage,
-  IonListHeader,
   IonGrid,
   IonRow,
   IonCol,
-  IonCard,
-  IonCardHeader,
-  IonCardTitle,
-  IonCardContent
+  IonSearchbar,
+  IonItem,
+  IonRouterLink,
+  IonButton,
+  IonList,
+  IonLabel,
+  IonSegment,
+  IonSegmentButton,
+  IonIcon,
 } from '@ionic/react';
-import data from "../../assets/data/data.json";
+import { people, document, documents } from 'ionicons/icons';
 import './Home.css';
-import parse from 'html-react-parser';
+import data from "../../assets/data/data.json";
+import { HomeTopic } from '../../components/home_topic';
 
 const Home: React.FC = () => {
-
-  const moemorizedCases = useMemo(() =>  data.cases, []);
-  const moemorizedTopics = useMemo(() =>  data.topics, []);
+  const guides = data.topics;
 
   return (
     <IonPage>
-      <IonContent>
-        <IonListHeader>
-          Highlights
-        </IonListHeader>
-        <IonGrid>
-          <IonRow>
-            {moemorizedTopics.filter((o) => o.highlighted === true).map((highlight, index) => (
-              <IonCol size="12" size-sm="6" size-xl="4" key={index}>
-                <IonCard routerLink={"guides/" + highlight.id}>
-                  <IonCardHeader>
-                    <IonCardTitle>{highlight.title}</IonCardTitle>
-                  </IonCardHeader>
-                  <IonCardContent>{parse(highlight.snippet)}</IonCardContent>
-                </IonCard>
+      <IonContent className="ion-padding">
+        <h2>South Africa's Constitution</h2>
+        <IonSearchbar placeholder="Find guides, cases or sections..." onIonChange={(e) => (e)}></IonSearchbar>
+        <IonItem>
```

Comment:
IonItem doesn't make sense outside of IonList -- I think you can simply drop IonItem.